src/main/dumpxdr.cpp

```diff
@@ -96,8 +97,10 @@ dumpxdr(std::string const& filename)
                                  xdr_strerror(errno));                         \
     } while (0)
 
+static std::string readFile(const std::string& filename, bool base64 = false);
```

Comment:
It's just for the default argument.  But I don't feel strongly about it for a function that is only called in a few places.